packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -1,26 +1,28 @@
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/Chart.yaml packages/rancher-gatekeeper/charts/Chart.yaml
 --- packages/rancher-gatekeeper/charts-original/Chart.yaml
 +++ packages/rancher-gatekeeper/charts/Chart.yaml
-@@ -1,6 +1,6 @@
+@@ -1,10 +1,17 @@
  apiVersion: v1
- description: A Helm chart for Gatekeeper
+ appVersion: v3.1.0
+-description: A Helm chart for Gatekeeper
++description: Policy-based control for cloud native environments
```

Comment:
We should reference the upstream chart here to give them credit, i.e. `Modifies the upstream <name> chart that provides policy-based control for cloud native environments`